src/traveler.js

```diff
@@ -0,0 +1,47 @@
+class Traveler {
+  constructor(name, trips) {
+    // this.id = data.id;
+    this.name = name;
+    // this.travelerType = data.travelerType
+    this.trips = trips;
+    // this.destinations = data.destinations
+  }
+  presentTrips() {
+    let currentDay = Date.now()
+    return this.trips.filter(trip => {
+      return trip.date === currentDay
+    })
```

Comment:
A word for thought, a user could be considered to be on a present trip if the current day is between when the trip started and when the date of the duration trip has ended as well.